src/main/java/uk/nhs/hee/tis/trainee/credentials/api/IssueResource.java

```diff
@@ -83,8 +87,10 @@ ResponseEntity<String> issuePlacementCredential(
       @Validated @RequestBody PlacementDataDto dataDto,
       @RequestParam(required = false) String state) {
     log.info("Received request to issue Placement credential.");
-    PlacementCredentialDto credentialDto = mapper.toCredential(dataDto);
-    Optional<URI> credentialUri = service.startCredentialIssuance(token, credentialDto, state);
+    BiFunction<TisDataDto, UUID, CredentialDto> mappingFunction = (data, id) -> mapper.toCredential(
```

Comment:
One of the few other places we use BiFunctions is in... tis-generic-upload :trollface: